### PR TITLE
Add thread template for problem 383. Ransom Note

### PR DESCRIPTION
**Pull request by miorel**

### CODE REVIEW

File: ransom-note.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## Checklist for a good session:
+
+* controlled the clock
+  * started on time
+  * ended on time
+* fostered a positive learning environment
+  * led the discussion with enthusiasm
+  * involved the audience
+  * confirmed that people understand the code being shown
+
```
Comment (Kratosmsn11) [inline ransom-note.md:26]: - cover the solution thats uses only a single array to store the occurrences of each character
- cover the solution using frequency map, only using one iteration. If time permits cover solution in other languages, preferably TypeScript/Python.
- if time permits, cover the solutions using some kind of built-in function.

File: ransom-note.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+TODO: add more
```
Comment (Kratosmsn11) [inline ransom-note.md:27]: - let 1 person from the audience share their code
- walkthrough of the solution using array to store the occurrences (possbly with the help of the volunteer sharing their screen)
- cover the frequency map solution
- if time permits, walk through the built-in function solution, else at the least mention it.

**Review by Kratosmsn11 — CHANGES_REQUESTED** [review]
I have added some comments with potential plan of action for the session on 9/15. Please review the following changes, and let me know if there should be anymore pointers to be covered.

File: ransom-note.md (hunk 1 above)
Comment (miorel) [reply ransom-note.md:26]: > cover the solution thats uses only a single array to store the occurrences of each character

Do you mean the solution that uses an array of size 26? That seems like a nice bonus optimization. I wouldn't consider that required for the "good" solution.

Comment (miorel) [reply ransom-note.md:26]: > If time permits cover solution in other languages, preferably TypeScript/Python.

That implies that you didn't start with TypeScript or Python to begin with 😛

**Review by Kratosmsn11 — APPROVED** [review]
Looks good to me!

**Pull request merged**